react-backend/routes/cbauthentication/checkCB.js

```diff
@@ -31,31 +31,34 @@ router.post('/', (req, res, next) => {
       validator.isEmail(data.formEmail) &&
       validator.isAlpha(org_name, ['en-GB'])
     ) {
-      getCBAlreadyExists(data.formEmail, (error, result) => {
-        if (error) {
+      getCBAlreadyExists(data.formEmail)
+        .then((result) => {
+          if (result.rows[0].exists === true) {
```

Comment:
I think this will change to just `if (result)`

You could even change the variable name to `cbExists` for clarity

Reply:
Also all the others below :+1: